### DISCUSSION TITLE
too many images to scan but it takes too long time, how can i do

### DISCUSSION THREAD

**Discussion by taczc64** [aquasecurity/trivy#1720] [Q&A] [answered]
trivy running in parallel takes same time as series run because of boltDB,  do we have good way to increase concurrent scan task? at first I think server/client mode  to handle this problem, but infact it's useless, because server have only one db file.

**Comment by afdesk** [comment, marked as answer]
@taczc64 I'd try to use `redis` for cache in client/server mode.
I did a test:
```sh
$ docker run -d --rm -p 6379:6379 redis:latest
$ trivy -d server --cache-backend redis://localhost:6379 --listen localhost:8080
```
and run 2 tests from 2 terminals:
```sh
$ trivy -d client --remote http://localhost:8080 elasticsearch:7.14.2
```
```sh
$ trivy -d client --remote http://localhost:8080 elasticsearch:5.6.13
```

**Reply by taczc64** [reply]
thanks, it's helpful
